### PR TITLE
Documentation: Changing index page global elements listing layout

### PR DESCRIPTION
**Pull request by Garneauma**
Layout of Sites and global functionality changed to match Templates and Components layout.

### CODE REVIEW

**Review by Ricokola — APPROVED** [review]

File: index-en.md [hunk 1]
```diff
@@ -295,33 +295,90 @@ css:
 </div>
 <div class="container">
 	<h2 id="sitesglobal">Sites and global functionality</h2>
-	<ul>
+	<ul class="row list-unstyled wb-eqht-grd wb-filter mrgn-tp-md"data-wb-filter='{ "selector": ">li" }'>
```
Comment (duboisp) [inline index-en.md:298]:
```suggestion
	<ul class="row list-unstyled wb-eqht-grd wb-filter mrgn-tp-md" data-wb-filter='{ "selector": ">li" }'>
```

File: index-fr.md [hunk 2]
```diff
@@ -294,34 +294,91 @@ css:
     </div>
 </div>
 <div class="container">
-	<h2 id="sitesglobal">Fonctionnalités global et de sites</h2>
-	<ul>
+	<h2 id="sitesglobal">Fonctionnalités globales et de sites</h2>
+	<ul class="row list-unstyled wb-eqht wb-filter mrgn-tp-md"data-wb-filter='{ "selector": ">li" }'>
```
Comment (duboisp) [inline index-fr.md:298]:
```suggestion
	<ul class="row list-unstyled wb-eqht wb-filter mrgn-tp-md" data-wb-filter='{ "selector": ">li" }'>
```

**Review by duboisp — CHANGES_REQUESTED** [review]
Good to go after the 2 little typo.

**Comment by Garneauma** [comment]
> Good to go after the 2 little typo.

C'est fait :)

**Review by duboisp — APPROVED** [review]

**Pull request merged**